table.go

```diff
@@ -465,13 +465,15 @@ coalesceLoop:
 			}
 			itMore, err := it.onValue(dims, itVals)
 			if err != nil {
+				log.Errorf("Error while iterating: %v", err)
 				return false, err
 			}
 			if !itMore {
 				// This iteration doesn't want any more data, stop feeding it
 				delete(remainingIterations, i)
+			} else {
+				more = true
```

Comment:
@forkner Sorry, I completely missed your original point. Added an else, which I think addresses the problem.